### PR TITLE
Added users import from Active Directory

### PR DESCRIPTION
**Pull request by wivern**
closes OHDSI/WebAPI#506

### CODE REVIEW

File: js/modules/WebAPIProvider/UsersAPI.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+define(function (require, exports) {
```
Comment (pavgra) [inline js/modules/WebAPIProvider/UsersAPI.js]: We agreed to services in `js/services` folder

File: js/modules/WebAPIProvider/UsersAPI.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+  const $ = require('jquery');
+  const config = require('appConfig');
+  const authApi = require('webapi/AuthAPI');
+
+  function getAuthenticationProviders() {
+    return $.ajax({
```
Comment (pavgra) [inline js/modules/WebAPIProvider/UsersAPI.js]: Use `services/http`

File: js/pages/configuration/users-import/components/atlas-roles.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<div>
+    <table width="100%" cellspacing="0" data-bind="dataTable:{
+                            data: roles(),
+                            options: {
+                                        dom: 'Bliprt',
```
Comment (pavgra) [inline js/pages/configuration/users-import/components/atlas-roles.html]: What is this for?

File: js/providers/Users.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
+define(function (require, exports) {
```
Comment (pavgra) [inline js/providers/Users.js]: Why is this separate from `js/modules/WebAPIProvider/UsersAPI.js`? + same comment about `services` folder

File: js/providers/Users.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  function getAuthenticationProviders() {
+    return $.ajax({
+      url: config.webAPIRoot + 'user/providers',
+      method: 'GET',
+      error: authApi.handleAccessDenied,
+    });
+  }
+
+  function searchGroups(provider, search) {
+  	return $.ajax({
```
Comment (pavgra) [inline js/providers/Users.js]: use `http` service

File: js/pages/configuration/users-import/components/atlas-roles.html (hunk 3 above)
Comment (wivern) [reply js/pages/configuration/users-import/components/atlas-roles.html]: It`s a wondering feature of [datatables](https://datatables.net/reference/option/dom),
for example, B means Buttons

**Pull request merged**